Gruntfile.js

```diff
@@ -54,11 +54,24 @@ module.exports = function(grunt) {
       options: {
         sourceMap: true
       },
-      my_target: {
+      minned: {
         files: {
           'build/fxpay.min.js': libFiles,
         }
+      },
+      debug: {
+        options: {
+          beautify: true,
```

Comment:
is there a way to have it use two space tabs so it looks more like the original source? Maybe that doesn't matter though.